engine/lib/luajit-ffi-gen/src/impl_item/generate_rust.rs

```diff
@@ -90,15 +90,15 @@ impl ImplInfo {
     // Note: We return a list of token streams here, because a single parameter can generate multiple parameters in the wrapper function.
     fn gen_wrapper_param(&self, param: &ParamInfo) -> Vec<TokenStream> {
         let name_ident = format_ident!("{}", param.name);
-        let param_type = param.ty.as_ffi(&self.name).rust;
+        let param_type = param.ty.as_ffi(&self.name).0;
         let param_type_tokens: TokenStream =
             param_type.parse().expect("Unable to parse Rust FFI type");
```

Comment:
I think that was me who added expect and panic in the generators but there should be more correct way to do it. Like in the parsing part we return an error. So probably for generation part there is a similar way.
But that's for later.